lib/suspenders/app_builder.rb

```diff
@@ -309,6 +309,76 @@ def setup_stylesheets
                 "app/assets/stylesheets/application.scss"
     end
 
+    def install_bourbon_n_friends
+      inject_into_file(
+        'Gemfile',
+        %{\ngem 'bourbon'\ngem 'neat'},
+        after: Regexp.new("gem 'rails', '#{Suspenders::RAILS_VERSION}'")
+      )
+      bundle_command 'install'
+
+      inject_into_file(
+        'app/assets/stylesheets/application.scss',
+        %{\n@import "bourbon";\n@import "neat";},
+        after: /@charset "utf-8";/
```

Comment:
Should also bring this back: https://github.com/philosophie/suspenders/commit/9fe8e01fd796ab92d5309bf306c76f6b595ec494